tensorflow/python/ops/rnn_cell_impl.py

```diff
@@ -13,7 +13,15 @@
 # limitations under the License.
 # ==============================================================================
 
-"""Module implementing RNN Cells."""
+"""Module implementing RNN Cells.
+
+This module provides a number of basic commonly used RNN cells, such as LSTM
+(Long Short Term Memory) or GRU (Gated Recurrent Unit), and a number of
+operators that allow adding dropouts, projections, or embeddings for inputs.
+Constructing multi-layer cells is supported by the class `MultiRNNCell`, or by
+calling the `rnn` ops several times. Every `RNNCell` must have the properties
+below and implement `__call__` with the following signature.
```

Comment:
The text isn't quite right for a module docstring, and not quite right for a class docstring. Can you disentangle the two?